core/admin.py

```diff
@@ -150,6 +165,12 @@ def data_link(self, obj):
 
     data_link.allow_tags = True
 
+    def formfield_for_dbfield(self, db_field, **kwargs):
```

Comment:
The `ScoreAlgorithmAdmin` class is nearly identical to the `AlgorithmAdmin` class.  Although it doesn't have to do directly with this PR other than that you are adding to it, we can replace the entire `ScoreAlgorithmAdmin` class with:

```
@admin.register(models.ScoreAlgorithm)
class ScoreAlgorithmAdmin(AlgorithmAdmin):
    list_display = ('__str__', 'task', 'creator', 'created', 'active', 'data_link')
```
to reduce code duplication.